neuron/src/lib/Neuron/Zettelkasten/Zettel/Meta.hs

```diff
@@ -45,25 +50,38 @@ instance FromYAML Meta where
 --     mapping
 --       [ "title" .= title,
 --         "tags" .= tags,
---         "date" .= date
+--         "created" .= date
 --       ]
 
-instance FromYAML Day where
+instance FromYAML LocalTime where
   parseYAML =
     parseZettelDate <=< parseYAML @Text
 
-instance ToYAML Day where
+instance ToYAML LocalTime where
   toYAML =
-    toYAML . formatZettelDate
+    toYAML . formatZettelLocalTime
 
--- | The format in which we decode and encode zettel dates.
-zettelDateFormat :: String
-zettelDateFormat = "%Y-%m-%d"
+-- | The default format in which we decode and encode zettel dates.
+defaultZettelDateFormat :: String
+defaultZettelDateFormat = "%Y-%m-%dT%H:%M"
+
+-- | All formats of user-edited tags 'created', that can be parsed.
+zettelDateFormats :: [String]
+zettelDateFormats = ["%Y-%m-%d", defaultZettelDateFormat]
+
+formatZettelLocalTime :: LocalTime -> Text
+formatZettelLocalTime =
+  toText . formatTime defaultTimeLocale defaultZettelDateFormat
 
 formatZettelDate :: Day -> Text
 formatZettelDate =
-  toText . formatTime defaultTimeLocale zettelDateFormat
+  toText . formatTime defaultTimeLocale "%Y-%m-%d"
```

Comment:
put `"%Y-%m-%d"` in a variable, `defaultZettelDayFormat`.

and rename `defaultZettelDateFormat` to `defaultZettelDateTimeFormat`

Reply:
Actually, just create an ADT:

```haskell
data ZettelDateFormat
  = ZettelDateFormat_Day
  | ZettelDateFormat_DateTime

zettelDateFormat :: ZettelDateFormat -> String 
zettelDateFormat = \case 
  ZettelDateFormat_Day -> "%Y-%m-%d"
  ...
```